Library/Homebrew/global.rb

```diff
@@ -30,7 +30,13 @@
 HOMEBREW_USER_AGENT_CURL = ENV["HOMEBREW_USER_AGENT_CURL"]
 HOMEBREW_USER_AGENT_RUBY = "#{ENV["HOMEBREW_USER_AGENT"]} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
 
-HOMEBREW_CURL_ARGS = "-f#RLA"
+HOMEBREW_CURL_ARGS = %W[
+  --fail
+  --progress-bar
+  --remote-time
+  --location
+  --user-agent #{HOMEBREW_USER_AGENT_CURL}
+]
```

Comment:
One small nit here and something I forgot in my previous suggestion: Maybe append a `.freeze`? It would both slightly benefit performance and—more importantly—prevent us from accidentally modifying this constant. (I believe RuboCop would have suggested this, too.)